main.go

```diff
@@ -54,26 +53,16 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+	pflag.String("metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	pflag.Bool("enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(&opconfig.HelmClientRepoCachePath, "helm-client-repo-cache-path",
+	pflag.String("helm-client-repo-cache-path",
```

Comment:
I think it would be beneficial to use constants for the flag names, this at least prevents a simple typo from breaking the flag parsing.

Reply:
:+1: Added and moved to the `controllers/config`-package.